db/schema.rb

```diff
@@ -0,0 +1,92 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_03_17_064629) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "foods", force: :cascade do |t|
+    t.string "name"
+    t.string "measurement_unit"
+    t.integer "price"
+    t.integer "quantity"
```

Comment:
{OPTIONAL]
 Kindly note that `quantity` is not present in the entity diagram for the FOOD model. Kindly update the schema 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/226117193-3fa67bac-44b3-4b07-b8b1-82118f73b372.png)